packages/swingset-runner/src/graphDisk.js

```diff
@@ -13,4 +14,12 @@ export async function main() {
   );
 }
 
-main();
+process.exitCode = 1;
+main().then(
+  () => {
+    process.exitCode = 0;
+  },
+  error => {
+    console.error(error);
+  },
+);
```

Comment:
Yes. I read this suggestion @bmeck recommended this pattern the other day and I was curious how it would socialize through review. It might be worthwhile to capture a utility to conduct this dance.